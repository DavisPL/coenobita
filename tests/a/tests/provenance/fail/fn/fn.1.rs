#![allow(unused_variables)]
#![allow(unused_assignments)]

#[cnbt::provenance((*,*) fn((b,bin)) -> (b,bin))]
fn id(x: i32) -> i32 {
    x + 1
}

fn main() {
    #[cnbt::provenance((b,bin))]
    let x = id(b::value());
}