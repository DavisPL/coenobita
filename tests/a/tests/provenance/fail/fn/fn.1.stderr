error: expected (b,bin), found (bin,bin)
 --> tests/provenance/fail/fn/fn.1.rs:6:5
  |
6 |     x + 1
  |     ^^^^^
